voice/src/main/kotlin/udp/GlobalVoiceUdpSocket.kt

```diff
@@ -40,12 +45,16 @@ public object GlobalVoiceUdpSocket : VoiceUdpSocket {
         globalVoiceSocketLogger.trace { "discovering ip" }
 
         send(packet(address) {
+            writeShort(REQUEST)
+            writeShort(70) // the length of the rest of the message
             writeInt(ssrc)
-            writeFully(ByteArray(66))
+            writeFully(EMPTY_DATA)
         })
 
         return with(receiveFrom(address).packet) {
-            discard(4)
+            require(readShort() == RESPONSE) { "did not receive a response." }
+            discardExact(6) // length (2) + ssrc (4)
```

Comment:
shouldn't this be checked too instead of being discarded?

Reply:
we could check the length to be 66, could give a better error message in the off chance discord messes up. ssrc would stay discarded though as in the previous behavior. theres nothing to do with it here.

Reply:
yeah, whatever, probably not worth the effort